library/classes/rulesets/PQRS/reports/PQRS_0007/Numerator2.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * PQRS Measure 0007 -- Numerator 2
+ *
+ * Copyright (C) 2016      Suncoast Connection
+ * @package PQRS_Gateway 
+ * @link    http://suncoastconnection.com
+ * @author  Bryan lee <bryan@suncoastconnection.com>
+ * @author  Art Eaton <art@suncoastconnection.com>
+ */
+ 
+class PQRS_0007_Numerator2 extends PQRSFilter
+{
+    public function getTitle()
+    {
+        return "Numerator 2";
+    }
+
+    public function test( PQRSPatient $patient, $beginDate, $endDate )
+    {
+		    $query =
+"SELECT COUNT(b1.code) as count".  
+" FROM billing AS b1".
+" JOIN form_encounter AS fe ON (b1.encounter = fe.encounter)".
+" WHERE b1.pid = ? ".
+" AND fe.date BETWEEN '".$beginDate."' AND '".$endDate."' ".
+" AND b1.code ='4008F' AND b1.modifier ='';";
+//8P is hard fail
```

Comment:
Yes.  the 8P modifier is a hard fail for this measure.  This note indicates a flag for expanding an additional feature to reflect additional reporting rates, and differentiate hard fails (had a code, were using CPT2 codes) vs a fail that is "not found, reporting not complete".